actionmailbox/test/unit/mailbox/routing_test.rb

```diff
@@ -28,4 +28,9 @@ class ActionMailbox::Base::RoutingTest < ActiveSupport::TestCase
       assert_equal "Discussion: Let's debate these attachments", $processed
     end
   end
+
+  test "mailbox_for" do
+    mail = create_inbound_email_from_fixture "welcome.eml", status: :pending
```

Comment:
Prefer to name variables that contain `ActionMailbox::InboundEmail` objects `inbound_email`. `mail` is used for `Mail::Message` instances.